src/openforms/prefill/api/urls.py

```diff
@@ -1,8 +1,29 @@
 from django.urls import path
 
-from .views import PluginAttributesListView, PluginListView
+from .views import (
+    PluginAttributesListView,
+    PluginListView,
+    PluginObjectsAPIAttributesListView,
+    PluginObjectsAPIObjecttypeListView,
+    PluginObjectsAPIObjecttypeVersionListView,
+)
 
 urlpatterns = [
+    path(
+        "plugins/objects-api/objecttypes",
+        PluginObjectsAPIObjecttypeListView.as_view(),
+        name="prefill-objects-api-objecttype-list",
+    ),
+    path(
+        "plugins/objects-api/objecttypes/<uuid:objects_api_objecttype_uuid>/versions",
+        PluginObjectsAPIObjecttypeVersionListView.as_view(),
+        name="prefill-objects-api-objecttype-version-list",
+    ),
+    path(
+        "plugins/objects-api/objecttypes/<uuid:objects_api_objecttype_uuid>/versions/<int:objects_api_objecttype_version>/attributes",
+        PluginObjectsAPIAttributesListView.as_view(),
+        name="prefill-objects-api-objecttype-attribute-list",
+    ),
     path("plugins", PluginListView.as_view(), name="prefill-plugin-list"),
     path(
         "plugins/<slug:plugin>/attributes",
```

Comment:
can you please separate these URL patterns in core/shared patterns + plugin specific patterns, just like we do with registrations: 
https://github.com/open-formulieren/open-forms/blob/beacb481e2dc31ec5fc9cfe56c157420b1f42d80/src/openforms/registrations/api/urls.py#L14

And please do make use of the `include` for urls and define the actual plugin URLs/views inside the `objects_api` package.